pkg/ui/app/containers/nodeGraphs.tsx

```diff
@@ -151,6 +151,30 @@ class NodeGraphs extends React.Component<NodeGraphsProps, {}> {
               </Axis>
             </LineGraph>
 
+            <LineGraph title="Backend Statement Execution Latency"
+                       subtitle="(Max Per Percentile)"
+                       tooltip={`The latency of backend statements executed over 10 second periods.
+                                 Percentiles are calculated on each node.`}
```

Comment:
You'll notice the other tooltips have a substitution token `${specifier}`, which gets translated into either "across all node" or "on node X" depending on the currently selected view.  That should be present somewhere in the tooltip.